ted_adapter/test_adapter.py

```diff
@@ -229,13 +247,20 @@ def main():
         # TODO return json string
         failed_cases = 1
 
+    try:
+        failed_cases_fei = adapter.run_fei_tests()
```

Comment:
it is not failed_cases, it is just error code now. you can use one variable for both scripts.